server/controllers/student/quizeDetails.ts

```diff
@@ -0,0 +1,15 @@
+import { NextFunction, Request, Response } from 'express';
+import { quizDetails } from '../../database/queries';
+import { CustomError } from '../../errors';
+
+export default async ({ params, cookies }:Request, res:Response, next:NextFunction) => {
```

Comment:
What is the purpose of this controller?
and why did you check for the token here? it's not the job of this controller to check if the user is authorized or not!